app/src/androidTest/java/com/simplemobiletools/calculator/MoneyActivityTest.java

```diff
@@ -84,6 +85,23 @@ public void taxWithoutGeoLocTest(){
         checkResult("25.99");
     }
 
+    @Test
+    public void testTipDialogDisplayed() {
```

Comment:
This test seems a bit redundant to me

Reply:
I think we should test it regardless. For example, if the actual tipping test fails and this one doesn't, we can conclude that the dialog is working. If both tests fail, we can conclude that the problem lies with the dialog.

Reply:
more tests is better than no test

Reply:
Fair enough